pyaedt/edb.py

```diff
@@ -309,7 +310,8 @@ def open_edb_inside_aedt(self, init_dlls=False):
             hdl = Convert.ToUInt64(self.oproject.GetEDBHandle())
             self._db = self.edb.Database.Attach(hdl)
             self._active_cell = self.edb.Cell.Cell.FindByName(self.db, self.edb.Cell.CellType.CircuitCell, self.cellname)
-            self.builder = self.layout_methods.GetBuilder(self.db, self._active_cell, self.standalone, True)
+            dllpath=os.path.join(os.path.abspath(os.path.dirname(__file__)), "dlls", "EDBLib", "DataModel.dll")
```

Comment:
You may consider pulling out the EDBLib path and make it a global variable in `edb.py`

Reply:
got it. I'll do in another PR after vacation